script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -895,23 +907,23 @@ I'm counting on you, Shiki! Let's fight together!"
 [sha:b6387fdece208d1e86a37b7e648715b190fcdb71]{
 -- Page 497, Offset 15992.
 -- 弾むような笑みを浮かべるアルクェイド。
-Arcueid brightens up into a full-faced smile.
+Arcueid is positively beaming.
 }
 [sha:fca4b6f43ea4ccff70a9b59678c75dba62979fe1]{
 -- Page 497, Offset 15993.
 -- ……頼りにするって、俺なんかいなくてもコイツひとりで充分だと思うけど……まあ、そう言ってくれるならやり甲斐はある。
-And who are you counting on, anyway...?
- I'm pretty sure she'd be totally fine without me.
- Though, her putting it like that does make it feel more worth doing.
+She's counting on me...?
+ I get the feeling that if I weren't here she'd be totally fine on her own.
```

Comment:
> She's counting on me...?
>  I get the feeling that if I weren't here she'd be totally fine on her own.

Both of these feel way better in the original. The second in particular is just unnecessarily long-winded and redundant for no particular reason.